### DISCUSSION TITLE
How would I use this for my use case

### DISCUSSION THREAD

**Discussion by walksanatora** [bolcom/libunftp#496] [Q&A] [answered]
how could Imake a FTP server where I can do things like
user `computer32` logs in with a password at `/home/ftp/computer/32/.ftppasswd` and gives them access to only `/home/ftp/computer/32`
user `computer16` logs in with a password at `/home/ftp/computer/16/.ftppasswd` and gives them access to only `/home/ftp/computer/16`
user `disk32` logs in with a password at `/home/ftp/disk/32/.ftppasswd` and gives them access to only `/home/ftp/disk/32`
because when I look through crates.io all I can see if FTP clients

**Comment by hannesdejager** [comment, marked as answer]
You will need to implement an new Authenticator: https://github.com/bolcom/libunftp/blob/master/src/auth/mod.rs
